docs/content.md

```diff
@@ -535,151 +535,121 @@ view = api.content.get_view(
 
-You can get all parents of a content object (from immediate parent to the portal root) using the {func}`api.content.get_parents` function.
+The following example shows how to get the absolute path from the Zope root.
 
 ```python
 from plone import api
 portal = api.portal.get()
-team = portal['about']['team']
 
-# Get all parents
-parents = api.content.get_parents(obj=team)
+folder = portal['events']['training']
+path = api.content.get_path(obj=folder)
+assert path == '/plone/events/training'
 ```
 
-% invisible-code-block: python
-%
-% self.assertEqual(parents[0], portal['about'])
-
-You can filter parent objects by interface:
+The following example shows how to get the portal-relative path.
 
 ```python
-from plone import api
-from Products.CMFCore.interfaces import IFolderish
-portal = api.portal.get()
-team = portal['about']['team']
-
-# Get folder parents only
-folder_parents = api.content.get_parents(obj=team, interface=IFolderish)
+rel_path = api.content.get_path(obj=folder, relative=True)
+assert rel_path == 'events/training'
 ```
 
+If the API is used to fetch an object with the `relative` parameter set as `True`, and the object is outside the portal, it throws an `InvalidParameterError` error.
+
 % invisible-code-block: python
 %
-% self.assertEqual(folder_parents[0], portal['about'])
-
-You can also filter parents using a custom predicate function:
+% # Setup an object outside portal for testing error case
+% app = portal.aq_parent
+% app.manage_addFolder('outside_folder')
+%
+% # Test that getting relative path for object outside portal raises error
+% from plone.api.exc import InvalidParameterError
+% with self.assertRaises(InvalidParameterError):
+%     api.content.get_path(obj=app.outside_folder, relative=True)
 
 ```python
-from plone import api
-portal = api.portal.get()
-team = portal['about']['team']
-
-# Get only published parents
-def is_published(obj):
-    try:
-        return api.content.get_state(obj=obj) == 'published'
-    except:
-        return False
+from plone.api.exc import InvalidParameterError
 
-published_parents = api.content.get_parents(obj=team, predicate=is_published)
+# Getting path of an object outside portal raises InvalidParameterError
+try:
+    outside_path = api.content.get_path(
+        obj=app.outside_folder,
+        relative=True
+    )
+    assert False, "Should raise InvalidParameterError and not reach this code"
+except InvalidParameterError as e:
+    assert "Object not in portal path" in str(e)
 ```
 
-(content-get-closed-parent-example)=
+(content-iter-ancestors-example)=
 
-## Get closest parent
+## Iterate over object ancestors
 
-To get the closest parent object that matches certain criteria, use the {func}`api.content.get_closest_parent` function.
+To iterate over the ancestors in an object acquisition chain, use the {func}`api.content.iter_ancestors` function.
```

Comment:
```suggestion
To iterate over the ancestors in the object tree, use the {func}`api.content.iter_ancestors` function.
```

Reply:
Why not have a convenience method of `get_closest_parent` that gets the first iteration of `api.content.iter_ancestors` with either `[0]` or `next`? I'm not sure if it's performant with a large or deep hierarchy.